lib/components/util/page-title.ts

```diff
@@ -0,0 +1,48 @@
+import { connect } from 'react-redux'
+import { useEffect } from 'react'
+
+import { isBlank } from '../../util/ui'
+
+interface Props {
+  appTitle: string
+  title: string | string[]
+}
+
+const TITLE_SEPARATOR = ' | '
+
+/**
+ * Invisible component that sets the page title (passed as a single string or array of strings)
+ * appended with the app title.
+ */
+const PageTitle = ({ appTitle, title }: Props) => {
+  useEffect(() => {
```

Comment:
We want to avoid passing string ids dynamically, so that the formatjs CLI tool can pick up the string for reporting. That tool is useful in instances such as #813, where we use a `yarn` task to find unused string ids. Passing or constructing ids dynamically would result in the CLI tool to miss those instances.